_includes/social.html

```diff
@@ -5,7 +5,8 @@
   {% assign title = page.title %}
 {% endif %}
 <hr />
-<h3 id="social">Share This Page</h3>
+{% assign lang = page.lang || site.default_lang %}
```

Comment:
There is exactly the same code [at line 2](https://github.com/nscyclone/elixir-school/blob/18e45569ebb5397235bc3f7d50f0e525ad2ef1a7/_includes/social.html#L2).

How about moving this line at the top of this file, and remove the other line which does same work?